administration/src/cards/activation.ts

```diff
@@ -12,15 +12,17 @@ import { ApolloClient } from '@apollo/client'
 
 export async function activateCard(
   client: ApolloClient<object>,
-  activationCode: DynamicActivationCode,
+  activationCode: DynamicActivationCode | StaticVerifyCode,
   region: Region
 ) {
   const cardInfoHash = await hashCardInfo(activationCode.pepper, activationCode.info!)
   const expirationDay = activationCode.info!.expirationDay
+  const totpSecret =
+    activationCode instanceof DynamicActivationCode ? uint8ArrayToBase64(activationCode.totpSecret) : null
   const card: CardGenerationModelInput = {
     cardExpirationDay: expirationDay ?? null, // JS number can represent integers up to 2^53, so it can represent all values of an uint32 (protobuf)
     cardInfoHashBase64: uint8ArrayToBase64(cardInfoHash),
-    totpSecretBase64: uint8ArrayToBase64(activationCode.totpSecret),
+    totpSecretBase64: totpSecret,
```

Comment:
I think we should discuss here if we want to determine here whether: totpSecret == null means that the card is static.

There is some implicit knowledge involved here.

Reply:
I am not entirely sure, how we should handle it, but to me, the current approach is a bit too implicit.
I would be okay with having a single mutation, but adding an extra field like `codeType: STATIC | DYNAMIC` and additional backend validation checking `codeType == STATIC  <=>  totpSecret == NULL`.

Reply:
agreed.